packages/ExternalLink/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@paprika/external-link",
   "version": "2.0.19",
-  "description": "The ExternalLink component links to an external website.",
+  "description": "DEPRECATED - The ExternalLink component links to an external website.",
```

Comment:
Probably we should say somewhere that the `<Link isExternalLink>` is the equivalent so consumers can easily refactor their code.  Not sure if this is the right place, or if we can put it somewhere else?

Reply:
There's a section for author notes in [@paprika/pill](https://www.npmjs.com/package/@paprika/pill). 
How about this place? 🤔 (I have no idea how to edit the page though)

<img width="1000" alt="Screen Shot 2021-11-26 at 10 35 10 AM" src="https://user-images.githubusercontent.com/92059030/143620174-57124408-391e-469a-ac46-0294f20b242e.png">

Reply:
Done.  
https://www.npmjs.com/package/@paprika/external-link

I discovered that the package can be deprecated via the website, but the message needs to be set via the command line.